actions/workflows/st2_e2e_tests.yaml

```diff
@@ -69,43 +69,25 @@ workflows:
                     host: <% $.host_ip %>
                     env: <% $.st2_cli_env %>
                 on-success:
-                    - robot_docs_tests
-            robot_docs_tests:
+                    - pabot_docs_tests
+            pabot_docs_tests:
                 action: core.remote
                 input:
                     hosts: <% $.host_ip %>
                     env: <% $.st2_cli_env %>
-                    cmd: cd /tmp/st2tests && . venv/bin/activate && pybot robotfm_tests/docs/
-                    timeout: 500
-                on-success:
-                    - robot_mistral_tests
-            robot_mistral_tests:
-                action: core.remote
-                input:
-                    hosts: <% $.host_ip %>
-                    env: <% $.st2_cli_env %>
-                    cmd: cd /tmp/st2tests && . venv/bin/activate && pybot robotfm_tests/mistral/
-                    timeout: 300
-                on-success:
-                    - robot_chatopsCI_tests
-            robot_chatopsCI_tests:
-                action: core.remote
-                input:
-                    hosts: <% $.host_ip %>
-                    env: <% $.st2_cli_env %>
-                    cmd: cd /tmp/st2tests && . venv/bin/activate && pybot robotfm_tests/chatopsCI/
+                    cmd: "cd /tmp/st2tests && . ./venv/bin/activate && bash /tmp/st2tests/robot_parallel_tests.sh '--processes 3 robotfm_tests/docs/' 0"
```

Comment:
This looks good :+1:, considering the task is logically only one `robotfm_tests/docs/` and we split it.